app/views/vocab_sheets/_vocab_sheet_item.html.haml

```diff
@@ -4,6 +4,10 @@
       = hidden_field_tag "item[#{vocab_sheet_item.id}][id]", vocab_sheet_item.id, class: 'item_id'
       = link_to sign_url(vocab_sheet_item.sign_id), { class: "drawing vocab_sheet_drawing drawing vocab_sheet_drawing__size--#{size}" } do
         = image_tag "/images/signs/400-400/#{vocab_sheet_item.drawing}"
+        = link_to "/images/signs/400-400/#{vocab_sheet_item.drawing}" do
```

Comment:
should we use `image_url` to hook into the asset pipeline here?

Reply:
Thanks Brenda. I will check this out : }